.ci/Jenkinsfile

```diff
@@ -250,6 +250,37 @@ pipeline {
         }
       }
     }
+
+    /**
+    Daily snapshots.
+    */
+    stage('Daily snapshots') {
+      options { skipDefaultCheckout() }
+      environment {
+        DOCKER_REGISTRY = 'docker.elastic.co'
+        DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
+        GCS_ACCOUNT_SECRET = 'secret/observability-team/ci/snapshoty'
+      }
+      when { branch 'main' }
+      steps {
+        withGithubNotify(context: 'Publish snapshot packages') {
+          deleteDir()
+          unstash name: 'source'
+          withNodeJSEnv(version: readFile(file: "${BASE_DIR}/.nvmrc")?.trim()) {
+            dir(env.BASE_DIR) {
+              sh(label: 'generate tarball', script: 'npm run package:tarball')
+              snapshoty(
+                bucket: 'obtl-artifacts',
```

Comment:
Is "obtl" a typo? The usual abbreviation I'm used to seeing from the observability-robots team is "oblt". I see that you've used the same "obtl-artifacts" for the Python and PHP PRs.

Reply:
We made a small typo. I renamed the GCS bucket with the correct name.